### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/backup [hunk 1]
```diff
@@ -1,38 +1,28 @@
...
 
-ynh_backup --src_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
+ynh_backup "/etc/php/$php_version/fpm/pool.d/$app.conf"
 
 #=================================================
 # END OF SCRIPT
 #=================================================
 
-ynh_print_info --message="Backup script completed for AdminerEvo. (YunoHost will then actually copy those files to the archive)."
+ynh_print_info "Backup script completed for Adminer. (YunoHost will then actually copy those files to the archive)."
```
Comment (CodeShakingSheep) [inline scripts/backup]:
```suggestion
ynh_print_info "Backup script completed for $app. (YunoHost will then actually copy those files to the archive)."
```

File: manifest.toml [hunk 2]
```diff
@@ -33,7 +34,7 @@ ram.runtime = "50M"
 
     [install.path]
     type = "path"
-    default = "/adminerevo"
+    default = "/adminer"
```
Comment (CodeShakingSheep) [inline manifest.toml]:
```suggestion
    default = "/adminerevo"
```

File: manifest.toml [hunk 3]
```diff
@@ -43,9 +44,19 @@ ram.runtime = "50M"
...
 [resources]
+
+    [resources.sources]
+
+    [resources.sources.main]
+    url = "https://github.com/adminerevo/adminerevo/releases/download/v4.8.4/adminer-4.8.4.php"
+    sha256 = "e9a9bc2cc2ac46d6d92f008de9379d2b21a3764a5f8956ed68456e190814b149"
+    rename= "index.php"
+    in_subdir = false
+
```
Comment (CodeShakingSheep) [inline manifest.toml:52]: +1 for adding sources to manifest. Thanks for that.

File: doc/screenshots/screenshot.png
Comment (CodeShakingSheep) [inline doc/screenshots/screenshot.png:1]: Why did you remove the screenshot?

File: sources/with_root_access/adminer.css
Comment (CodeShakingSheep) [inline sources/with_root_access/adminer.css:1]: AdminerEvo still needs a CSS theme as far as I know. I initially used [my fork of Hydra](https://github.com/CodeShakingSheep/Hydra-Dark-Theme-for-Adminer) (which has a fix compared to the original Hydra). Hydra is a very modern and beautiful theme. It looks like this.
image

We could just copy the contents from here https://raw.githubusercontent.com/CodeShakingSheep/Hydra-Dark-Theme-for-Adminer/master/adminer.css

Would also be great to include this in the manifest. As it's a raw link I'm not sure how it would work in the manifest. Perhaps you know @ericgaspar ?

**Review by CodeShakingSheep — CHANGES_REQUESTED** [review]
Thanks for your improvements @ericgaspar ! Just added a few comments.

File: sources/with_root_access/adminer.css
Comment (ericgaspar) [reply sources/with_root_access/adminer.css:1]: AdminerEvo works without CSS theme
This is what it looks like : 
Capture d’écran 2024-08-29 à 17 25 13

File: doc/screenshots/screenshot.png
Comment (ericgaspar) [reply doc/screenshots/screenshot.png:1]: we need to find a AdminerEvo screenshot (cf image below)

File: sources/with_root_access/adminer.css
Comment (CodeShakingSheep) [reply sources/with_root_access/adminer.css:1]: Ah ok, good to know.

**Comment by ericgaspar** [comment]
!testme
